fleece/connexion.py

```diff
@@ -104,15 +106,22 @@ def call_api(self, event):
                     message=msg,
                 )
             elif 500 <= response.status_code < 600:
-                # This case is generally enountered if the API endpoint handler
-                # code does not conform to the contract dictated by the Swagger
-                # specification. This case will also trigger if the handler
-                # code explicitly returns a 5xx error.
-                self.logger.error(
-                    'Raising 5xx error',
-                    response=response_dict,
-                    http_status=response.status_code,
-                )
+                if response_dict['title'] == RESPONSE_CONTRACT_VIOLATION:
+                    # This case is generally enountered if the API endpoint
+                    # handler code does not conform to the contract dictated by
+                    # the Swagger specification.
+                    self.logger.error(
+                        RESPONSE_CONTRACT_VIOLATION,
+                        detail=response_dict['detail']
+                    )
```

Comment:
No, this will only handle one case where the API contract is violated. It doesn't show a stacktrace, but it will indicate some details about the violation.

https://github.com/racker/fleece/pull/40/commits/bb83b11c030f19191d43608ab4dfbb0f997b3f83 does more of what you're looking for (by logging details of an uncaught exception in your code--such as a KeyError).